generatebundlefile/hack/update_bundles.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+set -e
+set -o pipefail
+
+BASE_DIRECTORY=$(git rev-parse --show-toplevel)
+# Function to display usage information
+function usage() {
+  echo "Usage: $0 -e <environment>"
+  echo "  -e    Specify the environment (staging or prod)"
+  exit 1
+}
+
+# Function to check if required tools are installed
+function check_requirements() {
+  local required_tools=("aws" "jq" "yq")
+  for tool in "${required_tools[@]}"; do
+    if ! command -v "$tool" &>/dev/null; then
+      echo "Error: $tool is not installed or not in PATH" >&2
+      exit 1
+    fi
+  done
+}
+
+# Function to get the latest version from ECR
+function get_latest_version() {
```

Comment:
Should we cross verify this somehow instead of just looking at latest, like GIT_TAG or anything else?

Reply:
We can verify it using the git commit of a project in the build tooling repo as an additional check, though I believe latest would always give us whatever the latest was build using the codebuild pipeline.

Reply:
Yea I think we should add that check just to make sure we are not just doing it from the latest, especially in like situations where a new image was accidentally added with a different version.

Reply:
We are also planning to introduce further automation through PR bot, so planning to add this additional check along with that improvement